recipes/barcode_splitter/run_test.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
```

Comment:
I'm sorry. The reason is that our current Docker tests do not support additional files. These files, run_tests.sh in your case, are not included in the Docker container and therefor can not be executed. Sorry for the confusion.